packages/@rnw-scripts/doxysaurus/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@rnw-scripts/doxysaurus",
```

Comment:
private: true?

Reply:
We do not have 'private' for other packages in this folder. @NickGerleman do we need to set private or not?

Reply:
I would lean towards keeping this one public. We publish a lot of our tools, which enables usage in other branches or repos.

Reply:
I will keep it public then as the other packages.